### ISSUE TITLE
How to determine hallucinations?

### ISSUE THREAD

**Issue by petergreis** [hf-lin/ChatMusician#5]
Very nice work. You mentioned that there is a tendency to hallucinate. How is this determined? Can I get the probabilities per token out?

Many thanks
Peter

**Comment by hf-lin** [comment]
To get the probabilities per token, you can get the logits generated by the model. Maybe this will help. https://discuss.huggingface.co/t/announcement-generation-get-probabilities-for-generated-output/30075/32.
